launch/hsi_color_filter.launch

```diff
@@ -0,0 +1,36 @@
+<launch>
+  <arg name="node_name" default="hsi_color_filter" />
+
+  <arg name="image" default="image" doc="The image topic. Should be remapped to the name of the real image topic." />
```

Comment:
Thank you. Memo: default topic `image` is correct and image_publisher is removed from launch file.